### DISCUSSION TITLE
找不到这张图片识别失败的原因

### DISCUSSION THREAD

**Discussion by Stave604671** [PaddlePaddle/PaddleOCR#13596] [Q&A] [answered]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug
![out2](https://github.com/user-attachments/assets/2a58c519-1908-4077-b04a-db2b884835bc)

![image](https://github.com/user-attachments/assets/4cbceeb1-8cfd-43d0-bf4a-21967a4befe3)
这张图不知道为什么OCR结果出不来,dpi已经调到300了,是清晰度太差还是怎么样

### Environment

os macos-13.0
environment. pycharm
python.  3.10.14
paddleocr. 2.7.3
cuda.  none

### Minimal Reproducible Example

![out2](https://github.com/user-attachments/assets/9b8c49c6-a4ee-4e5c-8cfd-a41d6fbcaa4e)
from paddleocr import PPStructure, PaddleOCR
engine = PaddleOCR(use_angle_cls=True, lang="ch")
result = engine.ocr(crop_image)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by kerry-weic** [comment, marked as answer]
估计是在检测阶段就滤掉了

**Reply by SWHL** [reply]
是的，这种直接走识别就行

**Reply by Stave604671** [reply]
好吧,我看文档里面的描述还以为这个就是默认直接识别,刚看到源代码里面有一个参数控制是否启动检测
